### PR TITLE
Short guide to installation and testa Windows

### PR DESCRIPTION
**Pull request by anderspersson**
Added a short guide/tutorial for setup and testa on windows using mingw compiler.

Thank your for contributing to Nuitka!

!! Please check that you select the **develop branch** (details see below link) !!

Before submitting a PR, please review the guidelines:
https://github.com/kayhayen/Nuitka/blob/master/CONTRIBUTING.md

### PR Checklist
- [ ] Correct base branch selected? `develop` for new features and bug fixes too. If it's
      part of a hotfix, it will be moved to ``master`` during it.
- [ ] All tests still pass. Check the developer manual about ``Running the Tests``. There
      are Travis tests that cover the most important things however, and you are
      welcome to rely on those, but they might not cover enough.
- [ ] Ideally new features or fixed regressions ought to be covered via new tests.
- [ ] Ideally new or changed features have documentation updates.

### CODE REVIEW

**Comment by kayhayen** [comment]
Sorry, we broke develop tests, so Travis is going to be red, but don't worry or rebase tomorrow, no problem there. Looks pretty good to me.

File: README.rst [hunk 1]
```diff
@@ -209,6 +209,76 @@ under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
...
+|                              
+|      def main():                  
+|          print( talk("Hello World"))  
+|                              
+|      if __name__ == "__main__":   
+|          main()
+                       
+____
+
+3. testa it using **python hello.py**
```
Comment (kayhayen) [inline README.rst]: Stupid question, is `testa` a typo or swedish or ... :-)

**Comment by anderspersson** [comment]
Sorry Swedish for test or testing,
my tutorial was in Swedish so i translated it and made a miss there.
You och i fix it ?
/A

Den fre 15 feb. 2019 kl 22:41 skrev Kay Hayen <notifications@github.com>:

**Comment by kayhayen** [comment]
Nah, don't worry, I will make a few minor edits and formatting anyway.

**Pull request merged**